src/backend/base/langflow/graph/graph/base.py

```diff
@@ -914,13 +914,13 @@ async def build_vertex(
             else:
                 raise ValueError(f"No result found for vertex {vertex_id}")
             flow_id = self.flow_id
-            log_transaction(flow_id, vertex, status="success")
+            asyncio.create_task(log_transaction(flow_id, vertex, status="success"))
```

Comment:
You should probably remove these two calls here. I feel like these should be vertex builds instead, and we can add a tab in the logs table later.

Reply:
removed, thanks